OloEditor/src/EditorLayer.cpp

```
@@ -24,1110 +24,1100 @@
-		settings.m_CompressAssets = true;
-		settings.m_IncludeScriptModule = true;
-		settings.m_ValidateAssets = true;
-
-		// Reset progress and flags
-		m_BuildProgress.store(0.0f);
-		m_BuildCancelRequested.store(false);
-		m_BuildInProgress.store(true);
-
-		// Start async build task using std::async
-		m_BuildFuture = std::async(std::launch::async, [this, settings]() -> AssetPackBuilder::BuildResult {
+    static Font* s_Font;
```

Clang (14.0.6)

[warning] variable 's_Font' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)